bodhi/server/templates/master.html

```diff
@@ -202,7 +202,7 @@
           bodhi</a> is Free Software.
           Please <a href="https://github.com/fedora-infra/bodhi/issues">
           file issues</a>
-          if you have any problems.
+          if you have any problems. Read the <a href="https://bodhi.fedoraproject.org/docs">documentation</a>
```

Comment:
Let's add a period at the end of this sentence so it is consistent with the other two sentences on the line.